docs/docs/20-usage/50-environment.md

```diff
@@ -77,6 +77,7 @@ This is the reference list of all environment variables available to your pipeli
 | `CI_COMMIT_AUTHOR`               | commit author username                                                                                             |
 | `CI_COMMIT_AUTHOR_EMAIL`         | commit author email address                                                                                        |
 | `CI_COMMIT_AUTHOR_AVATAR`        | commit author avatar                                                                                               |
+| `CI_COMMIT_PRERELEASE`           | release is a pre-release (empty if event is not `release`)                                                         |
```

Comment:
I like that approach. Could we use sth similar for draft releases? Currently it will only handle release creation, right? Do you have any ideas how we could add update / delete in the long term if needed?